packages/sdk-node/src/utils.ts

```diff
@@ -26,7 +26,7 @@ export const loadClientEnv = () => {
 export const loadApiEnv = () => {
   return {
     secretKey: process.env.CLERK_SECRET_KEY || process.env.CLERK_API_KEY || '',
-    apiKey: process.env.CLERK_SECRET_KEY || process.env.CLERK_API_KEY || '',
```

Comment:
Are there usages of `apiKey` that assume it has been defaulted to `CLERK_SECRET_KEY`?

Reply:
i think it's a typo or an attempt to be "extra" safe that the env is used.